.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think. It's better to put it inside `package.json`. Fewer files inside the project root look easier and less verbose for new users.